docs/introduction/blockchain-representation.rst

```diff
@@ -2,51 +2,51 @@
     
   * **State**
     * “State” is a snapshot of all boxes that haven’t been opened yet. It represents the state at the current chain tip.
     
   * **Wallet**
     * The “Wallet” has two main functionalities:
-      1. It holds the Secret keys that belong to that specific Node.
+      1. It holds the secret keys that belong to that specific node.
       2. It keeps track of objects that are of interest to this specific node, e.g. received coins (output boxes whose secret keys are known by the node) and views of them (e.g. balances).      
   * **Memory Pool**
-    * The “Memory pool” is a list of transactions that are known to the node but have not made it to a Sidechain block yet.
+    * The “Memory pool” is a list of transactions that is known to the node but have not made it to a Sidechain block yet.
     
 Altogether these 4 objects represent a “NodeView.”
 
 NodeViewHelper
 ==============
 
-All communication between NodeView objects is controlled by NodeViewHolder, which also provides a layer of communication within the application for local data processing of Blocks, Transactions, Secrets, etc.
+All communication between NodeView objects is controlled by NodeViewHolder, which also provides a layer of communication within the application for local data processing of blocks, transactions, secrets, etc.
 
 In terms of customization, the History object is the only one that is fully controlled by the core and that in almost all circumstances does not need to be extended. It contains a ready-made implementation of the Latus consensus and of the Cross-Chain Transfer Protocol.
 
-The core logic of State, Wallet and Memory Pool can instead be extended by sidechain developers:
+The core logic of State, Wallet and Memory Pool objects can be extended by sidechain developers:
 
- * The “State” is a set of objects that are the result of processing all the previous blocks. These objects are needed to validate the next block, to allow the Node to efficiently verify, before applying a block, that all the defined rules have been respected by it. The “State” can be extended to keep track of new objects that can be useful to enforce additional rules that can be implemented in the application state interface.
+ * The “State” is the set of objects that result from processing all the previous blocks. These objects are needed to validate the next block, to allow the Node to efficiently verify, before applying a block, that all the defined rules have been respected by it. The “State” can be extended to keep track of new objects that can be useful to enforce additional rules that can be implemented in the application state interface.
 
- * The “Wallet” can be extended  through the ApplicationWallet interface, e.g. to change box ownership rules.
+ * The “Wallet” can be extended through the ApplicationWallet interface, e.g. to change box ownership rules.
```

Comment:
@zennermac  * The “wallet” can be extended through the ApplicationWallet interface, e.g. to change box ownership rules.